### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by tmzaragoza**

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+      res.json(result.rows[0]);
+    });
+});
+app.delete('/api/racers/:id', (req, res) => {
+  const body = req.body;
+  client.query(`
+    DELETE FROM racer
+    WHERE id = $1;
+  `,
+  [body.id])
```
Comment (grayson073) [inline server/server.js]:
```suggestion
  [req.params.id])
```

File: server/server.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  WHERE id = $5
+  RETURNING *;
+  `,
+  [body.name, body.age, body.teamId, body.pr, body.id])
+    .then(result => {
+      res.json(result.rows[0]);
+    });
+});
+app.delete('/api/racers/:id', (req, res) => {
+  const body = req.body;
```
Comment (grayson073) [inline server/server.js]:
```suggestion

```

File: app/src/services/api.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+            method: 'PUT',
+            headers: {
+                'Content-Type': 'application/json'
+            },
+            body: JSON.stringify(racer)
+        })
+            .then(response => response.json()); 
+    },
+    deleteRacer(id) {
+        return fetch (`/api/racer/${id}`, {
```
Comment (grayson073) [inline app/src/services/api.js]:
```suggestion
        return fetch (`/api/racers/${id}`, {
```
Forgot an 's'

**Comment by tmzaragoza** [comment]
Should have looked at this first!

**Pull request closed without merging**